plugin/trino-redis/src/main/java/io/trino/plugin/redis/RedisRecordCursor.java

```diff
@@ -300,24 +339,22 @@ private void fetchKeys()
                     log.debug("Scanning new Redis keys from cursor %s . %d values read so far", cursor, totalValues);
 
                     redisCursor = jedis.scan(cursor, scanParams);
-                    List<String> keys = redisCursor.getResult();
-                    keysIterator = keys.iterator();
+                    keys = redisCursor.getResult();
                 }
                 break;
                 case ZSET:
-                    List<String> keys = jedis.zrange(split.getKeyName(), split.getStart(), split.getEnd());
-                    keysIterator = keys.iterator();
+                    keys = jedis.zrange(split.getKeyName(), split.getStart(), split.getEnd());
                     break;
                 default:
-                    log.debug("Redis key of type %s is unsupported", split.getKeyDataFormat());
+                    log.warn("Redis key of type %s is unsupported", split.getKeyDataFormat());
```

Comment:
Reminder.

Reply:
Hi @ebyhr, I added this modification to the commit message, because this modification is minor, do you think this is ok?
![image](https://user-images.githubusercontent.com/29967142/165672444-d2bdf0db-2c4b-464a-b6dc-b0f3c7f50696.png)